test/e2e/operator_test.go

```diff
@@ -1291,6 +1299,80 @@ func TestInternalLoadBalancerGlobalAccessGCP(t *testing.T) {
 	}
 }
 
+// TestAWSResourceTagsChanged tests the functionality of updating AWS resource tags
+// in the infrastructure configuration and validates that the expected
+// awsLBAdditionalResourceTags is set correctly on the
+// loadBalancer service associated with the default Ingress Controller.
+//
+// This test is a serial test because it modifies the cluster infrastructure config and
+// therefore should not run in parallel with other tests.
+func TestAWSResourceTagsChanged(t *testing.T) {
+	if infraConfig.Status.Platform != "AWS" {
+		t.Skipf("test skipped on platform %q", infraConfig.Status.Platform)
+	}
+	if err := waitForIngressControllerCondition(t, kclient, 10*time.Second, defaultName, defaultAvailableConditions...); err != nil {
+		t.Errorf("did not get expected conditions: %v", err)
+	}
+	defaultIC := &operatorv1.IngressController{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: defaultName.Namespace,
+			Name:      defaultName.Name,
+		},
+	}
+	awsLBAdditionalResourceTags := "service.beta.kubernetes.io/aws-load-balancer-additional-resource-tags"
+
+	// Revert to original status
+	originalInfraStatus := infraConfig.Status
+	t.Cleanup(func() {
+		updateInfrastructureConfigStatusWithRetryOnConflict(configClient, func(infra *configv1.Infrastructure) *configv1.Infrastructure {
+			infra.Status = originalInfraStatus
+			return infra
+		})
+	})
```

Comment:
Something like this:
```suggestion
	}
        if err != nil {
             t.Logf("Unable to remove changes to the infraConfig: %v, possible corruption of test environment", err)
        }
	)
```

Reply:
Sure, updated.